### DISCUSSION TITLE
General thought about writing review

### DISCUSSION THREAD

**Discussion by guyzilberblum** [Cyfrin/security-and-auditing-full-course-s23#169] [General]
I have a question in the coure in  the first audit the password storge there is a mitigation explaind wich seems a bit too "fansy" to me, my question is can i write this in simple words it seems like it will make the developers understand better my explenations  its just seems not that useful to use such a high words of a "fansy" description if i can explain it in simple words why not  if im missing something you are free to let me know thanks

**Comment by drained69** [comment]
I think what a good review should include would be subjective to different people but then in my short time here I think the best of findiings do includes clear summary of the bug, POC, and recommendation.

**Comment by EngrPips** [comment]
There are no strict rules on how to write a report. Just make yourself clear enough and prove that the bug does affect the protocol
